CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ endif (NOT CMAKE_BUILD_TYPE MATCHES "Release")
 
 # Set beta version if this is a beta release series,
 # unset if this is a stable release series.
-set (PROJECT_BETA_RELEASE 1)
+set (PROJECT_BETA_RELEASE 0)
```

Comment:
Shouldn't this be commented out to be a stable version? I am not 100 percent sure...
```suggestion
# set (PROJECT_BETA_RELEASE 0)
```

Reply:
Tested and `set (PROJECT_BETA_RELEASE 0)` works too